data/json/items/chemicals_and_resources.json

```diff
@@ -890,5 +890,24 @@
     "weight": 212,
     "volume": 1,
     "to_hit": -2
-    }
+    },{
+    "type" : "COMESTIBLE",
```

Comment:
If it's not comestible, it should either not be an item at all until it gets a proper use, or should get an use here.

If it wasn't comestible, the only way to actually use it for anything would be to pour it on fire. But that doesn't happen often enough to warrant having it as an item.

I think mugling's comment was regarding stuff like salt water and bleach, not sewage.
Sewage looks relatively safe - unhealthy, horribly un-fun, but a desperate early game survivor forced into sewers could actually want to drink it. Similar to how eating paper worked before it was removed (though paper was nutrition and so more useful).
Drinking salt water and bleach is never a good idea, drinking sewage is better than being totally dehydrated.